SoftwareGuide/Latex/Introduction/Introduction.tex

```diff
@@ -267,9 +266,9 @@ \section{A Brief History of ITK}
 
 In 2002 the first official public release of ITK was made available. In
 addition, the National Library of Medicine awarded thirteen contracts to
-several organizations to extend ITK's capabilities. The NLM has funded
+several organizations to extend ITK’s capabilities. The NLM has funded
```

Comment:
Not sure this was intended. Usually the used apostrophe is neutral (typewriter apostrophe or vertical apostrophe) vs.  typographic apostrophe, right?

Reply:
Great catch -- LaTeX needs the old version.